src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts

```diff
@@ -211,7 +217,16 @@ export class SearchableDropdownComponent implements ControlValueAccessor, OnChan
             this.change.emit(Object.assign({}, this.value));
         }
 
-        this.searchPanelRef.hide();
+        this.toggleOverlayPanel();
+    }
+
+    /**
+     * Shows or hide the list of options.
+     * @param MouseEvent event
+     * @memberof SearchableDropdownComponent
+     */
+    toggleOverlayPanel($event?: MouseEvent): void {
```

Comment:
Conceptually this is weird, because the name is `toggle` and also the flag is the event which doesn't make much sense, but is not always toggling, shouldn't we replace this with `open` and `close` methods?